source/customizing-application/configuring-faceted-search.rst

```diff
@@ -129,7 +198,11 @@ Aggregation based on queries. One query will define one bucket in the aggregatio
   }
 
 
-Key like availableInViewService may not be available in the translations. Use the translation API to add your custom translation in the database.
+Key like ``availableInViewService`` may not be available in the translations. Use the translation API to add your custom translation in the database (see the admin console > Settings > Languages).
```

Comment:
I would update `admin console` to `Admin console`.

Should be added some formatting to the option names?